src/main/java/app/bpartners/geojobs/endpoint/rest/controller/ZoneTilingController.java

```diff
@@ -98,6 +100,7 @@ public TaskStatistic getTilingTaskStatistics(@PathVariable String id) {
 
   @PostMapping("/tilingJobs")
   public ZoneTilingJob tileZone(@RequestBody CreateZoneTilingJob createJob) {
+    communityZoneTilingJobProcessAuthorizer.accept(createJob);
```

Comment:
Add a TODO here. 
As I said, authorization handling must be made inside SecurityConf, not explicitly here inside the exposed Controller.